### DISCUSSION TITLE
Github Copilot stuck in disable mode

### DISCUSSION THREAD

**Discussion by serunos** [community/community#39918] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I disabled GitHub copilot and now I want to enable it back but there is no option to do so.
I'm on Visual Studio.
![image](https://user-images.githubusercontent.com/72872185/203755681-2527b299-3ae7-4f4f-b89f-7c158bde605e.png)
![image](https://user-images.githubusercontent.com/72872185/203755719-7125f13e-a69f-485b-a02b-e26b60b59f8b.png)

**Comment by ossanna16** [comment]
Hi there @serunos and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by jfullstackdev** [comment]
how about this 

https://github.com/community/community/discussions/10930

**Reply by serunos** [reply]
Hi,
This answer is for VS code, and not for Visual Studio.
Moreover, I can't interact with GitHub copilot because every option is disabled.

**Comment by Tovrick** [comment, marked as answer]
Just ran into this problem.  Had to create a keyboard shortcut to enable copilot globally (tools -> options -> keyboard --> Edit.Copilot.ToggleGlobally) .  Once enabled globally I restarted Visual Studio and all the copilot options were showing up again.

**Reply by serunos** [reply]
Thanks, it worked for me.

**Reply by danielo515** [reply]
Thanks,worked for me too

**Reply by thebitanpaul** [reply]
![Screenshot 2024-02-07 at 12 48 17 PM](https://github.com/community/community/assets/99794785/14b3fd11-e549-4e87-88ea-681ad84ab83b)

assign a keyboard shortcut to enable/disable the copilot, and it will work like a charm.

**Comment by divinity76** [comment]
Had the same problem today on "1.85.1, 2023-12-13", and a system reboot fixed it.. ¯\\\_(ツ)_/¯

**Comment by thebitanpaul** [comment]
Assign a keyboard shortcut to enable/disable the copilot, and it will work like a charm.

![Screenshot 2024-02-07 at 12 48 17 PM](https://github.com/community/community/assets/99794785/1c0779c7-6ba2-4de0-a838-388c5a4681ff)

**Reply by dylanrussellmd** [reply]
This also worked for me. What strange and annoying behavior.

**Comment by BellaBe** [comment]
This is just a bad dev experience, considering it is a paid feature. It would be great if somebody could fix this issue!
